cdk/lib/__snapshots__/frontend.test.ts.snap

```diff
@@ -124,9 +124,6 @@ Object {
           "Ref": "AMIFrontend",
         },
         "InstanceType": "t4g.small",
-        "MetadataOptions": Object {
-          "HttpTokens": "required",
-        },
```

Comment:
This is the part of the cloudformation that dictates if your instances will use `IMDSv1` or `IMDSv2`. Requiring tokens = using v2, so removing this is exactly what we want to see